### PR TITLE
Git commands by surya

### PR DESCRIPTION
**Pull request by surya-teja98**

### CODE REVIEW

File: git_and_linex_commands/git_commands.txt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+git reset
+Usage: git reset [file] - This command unstages the file, but it preserves the file contents.
+Usage: git reset [commit] - This command undoes all the commits after the specified commit and preserves the changes locally.
+Usage: git reset –hard [commit] - This command discards all history and goes back to the specified commit.
+
+git status
+Usage: git status - This command lists all the files that have to be committed.
+
+git rm
+Usage: git rm [file] - This command deletes the file from your working directory and stages the deletion.
```
Comment (avinash367) [inline git_and_linex_commands/git_commands.txt:35]: Add some more commands for removing branches

**Review by avinash367 — COMMENTED** [review]
Where is linux commands...?

File: git_and_linex_commands/git_commands.txt (hunk 1 above)
Comment (surya-teja98) [reply git_and_linex_commands/git_commands.txt:35]: yaa i will

**Review by kulkarni90 — COMMENTED** [review]
Good . practice  commands regularly.

**Review by avinash367 — COMMENTED** [review]
Good work

**Pull request merged**